docs/user-guide/personalidentifiableinformationidentificationandremoval.rst

```diff
@@ -241,7 +268,38 @@ For example:
 
 The ``LLMPiiModifier`` module can be invoked via the ``nemo_curator/scripts/llm_pii_redaction.py`` script which provides a CLI-based interface. To see a complete list of options supported by the script, execute:
 
-``llm_pii_redaction --help``
+.. code-block:: bash
+
+    llm_pii_redaction --help
+
+Example of using the non-async CLI tool:
+
+.. code-block:: bash
+
+    llm_pii_redaction \
+      --input-data-dir /path/to/input \
+      --output-data-dir /path/to/output \
+      --base_url "http://0.0.0.0:8000/v1" \
+      --api_key "your_api_key"
+
+Custom System Prompts
+~~~~~~~~~~~~~~~~~~~~~~
+
+When working with non-English text or when you want to customize how the LLM identifies PII entities, you can provide a custom system prompt. However, ensure that the JSON schema is included exactly as shown in the default system prompt.
+
+For reference, the default system prompt is:
+
+.. code-block:: text
+
+    "You are an expert redactor. The user is going to provide you with some text. 
+    Please find all personally identifying information from this text. 
+    Return results according to this JSON schema: {JSON_SCHEMA}
+    Only return results for entities which actually appear in the text. 
+    It is very important that you return the entity_text by copying it exactly from the input. 
+    Do not perform any modification or normalization of the text. 
+    The entity_type should be one of these: {PII_LABELS}"
```

Comment:
Not as important, but maybe specify that `PII_LABELS` is a list of strings, e.g., `name`, `ip_addres`, etc.